src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -8005,5 +8477,482 @@ static void Main()
 (100, 200, 300, 400, 500, 600, 700, (800, 900))
 ");
         }
+
+        [Fact]
+        public void PropertiesAsMembers_01()
+        {
+            var source = @"
+using System;
+class C
+{
+    static void Main()
+    {
+        var v1 = M1();
+        v1.P1 = 33;
+        System.Console.WriteLine(v1.P1);
+        System.Console.WriteLine(v1[-1, -2]);
+    }
+
+    static (int, int) M1()
+    {
+        return (1, 11);
+    }
+
+    static (int a2, int b2) M2()
```

Comment:
ToString is also "used" as it affects the list of members.

---
In reply to: [62357088](https://github.com/dotnet/roslyn/pull/11136#discussion_r62357088) [](ancestors = 62357088)